lib/web_ui/test/raw_keyboard_test.dart

```diff
@@ -748,11 +748,11 @@ DomKeyboardEvent dispatchKeyboardEvent(
 }) {
   target ??= domWindow;
 
-  final Function jsKeyboardEvent =
-      js_util.getProperty<Function>(domWindow, 'KeyboardEvent');
-  final List<dynamic> eventArgs = <dynamic>[
+  final Object jsKeyboardEvent =
+      js_util.getProperty<Object>(domWindow, 'KeyboardEvent');
```

Comment:
Should we do the `createKeyboardEvent()` (in `dom.dart`) pattern here as well? It doesn't have to be done in this PR though.